Makefile

```diff
@@ -1,21 +1,64 @@
 DOCKER_IMAGE := m2cgen
 DOCKER_RUN_ARGS := docker run --rm -it -v "$$PWD":"/m2cgen" $(DOCKER_IMAGE)
+TARGETS := ./m2cgen ./tests
 
+clean:
+	rm -rf ./m2cgen.egg-info ./dist ./build
+
+flake8:
+	flake8 $(TARGETS)
+
+isort:
+	isort $(TARGETS)
+
+isort-check:
+	isort $(TARGETS) --check-only
+
+test-api:
+	pytest -v tests/ --cov=m2cgen/ --cov-report=xml:coverage.xml --ignore=tests/e2e/
+
+package:
+	python setup.py bdist_wheel --plat-name=any --python-tag=py3 && \
+		python setup.py sdist
+
+publish: clean package
+	python -m twine upload dist/*
+
+install-requirements:
+	pip install -r requirements-test.txt
+
+install-develop:
+	python setup.py develop
+
+pre-pr: install-requirements flake8 isort test-api
+
+generate-code-examples: install-develop
+	python tools/generate_code_examples.py ./generated_code_examples
+
+download-codecov:
+	wget -q https://uploader.codecov.io/latest/linux/codecov -O codecov && \
+		chmod +x codecov
+
+run-codecov: download-codecov
+	./codecov -f coverage.xml -Z
 
 docker-build:
 	docker build -t $(DOCKER_IMAGE) .
 
 docker-test-all:
 	$(DOCKER_RUN_ARGS)
 
```

Comment:
I believe this command isn't used anymore.
```suggestion
```
And this one is duplicated (can't make a GitHub suggestion):
https://github.com/BayesWitnesses/m2cgen/blob/2447b98553fa28c8ab1c00ab7a716c3c11721a76/Makefile#L66

Reply:
Ah, good catch! Completely overlooked that, thanks!